### ISSUE TITLE
help: Unable to load devtools

### ISSUE THREAD

**Issue by ahoiroman** [nuxt/devtools#394]
### 📚 What are you trying to do?

I am using dev tools with 

```
export default defineNuxtConfig({
    devtools: {enabled: true},
//..
});
```

Now I am getting these errors:

<img width="915" alt="Bildschirmfoto 2023-08-17 um 12 54 45@2x" src="https://github.com/nuxt/devtools/assets/23051150/1e965804-2c95-4e4a-819b-e36d703f064d">

I am **not** able to reproduce this on stackblitz, so I guess there's something wrong with my local setup. But what could that be? Where do I have to look?

### 🔍 What have you tried?

I tried to reinstall all dependencies, deleted the package lock file and `node_modules`, making sure 

```
    "@nuxt/devtools": "latest",
```

is installed.

**Comment by ahoiroman** [comment]
I mentioned that this error appears, once `buildAssetsDir: '/_application/'` is set.

**Comment by cuebit** [comment]
This is resolved by #386. You can use the edge release or await the next stable release.
